ports/portmidi/portfile.cmake

```
@@ -34,14 +41,27 @@ vcpkg_build_cmake()
 file(INSTALL ${SOURCE_PATH}/pm_common/portmidi.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
 file(INSTALL ${SOURCE_PATH}/porttime/porttime.h DESTINATION ${CURRENT_PACKAGES_DIR}/include)
```

```suggestion
file(INSTALL "${SOURCE_PATH}/pm_common/portmidi.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
file(INSTALL "${SOURCE_PATH}/porttime/porttime.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
```
Please add double quotes for these paths.